src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -30,16 +29,14 @@
 (defn- handle-on-viewable-items-changed
   [{:keys [event active-category should-update-active-category?]}]
   (when should-update-active-category?
-    (let [viewable-item (-> (oops/oget event "viewableItems")
-                            transforms/js->clj
-                            first
-                            :item)
+    (let [viewable-item (some-> event
+                                (oops/oget "viewableItems")
+                                (aget 0)
+                                (.-item))
           header?       (and (map? viewable-item) (:header? viewable-item))
           section-key   (if header?
                           (:id viewable-item)
-                          (:id (emoji-picker.data/emoji-group->category (-> viewable-item
-                                                                            first
-                                                                            :group))))]
+                          (-> viewable-item first :group emoji-picker.data/emoji-group->category :id))]
```

Comment:
This function had a major improvement in performance! :smile: 
used the `time` function to wrap this piece of code, in develop it's taking ~22ms!! the less I got was 12ms and as much as 33ms.

Transfroming all values to only get the first item has a big impact in performance.
Now we are just extracting from that JS vector.
The time now for this function is  0.02ms ~ 0.03ms

Reply:
The `viewableItems` inside the event sometimes give out nil when the user scrolls through the list. But `some->` is a great choice to tackle that.

Reply:
@smohamedjavid  Yep, before adding `some->` I had that issue :sweat_smile: